CODEOWNERS

```diff
@@ -34,75 +34,75 @@
-/doc/ref_cert/RC1/chapters/chapter07.md        @scottsteinbrueck @fuqiao123
-/doc/ref_cert/RC1/chapters/chapter08.md        @scottsteinbrueck @fuqiao123
-/doc/ref_cert/RC1/chapters/chapter09.md        @scottsteinbrueck @fuqiao123
-/doc/ref_cert/RC1/chapters/chapter10.md        @scottsteinbrueck @fuqiao123
-/doc/ref_cert/RC1/chapters/chapter11.md        @scottsteinbrueck @fuqiao123
+/doc/ref_cert/RC1/chapters/chapter01.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter02.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter03.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter04.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter05.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter06.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter07.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter08.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter09.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter10.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter11.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
 
 
 # Reference Architectures 2
-/doc/ref_arch/kubernetes/chapters/chapter01.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter02.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter03.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter04.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter05.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter06.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter07.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter08.md        @scottsteinbrueck @rgstori
+/doc/ref_arch/kubernetes/chapters/chapter01.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter02.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter03.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter04.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter05.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter06.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter07.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter08.md        @scottsteinbrueck @rgstori @walterkozlowski
 
 
 # Reference Implementation 2
```

Comment:
@tomkivlin no longer engaged. Should we add both Rihab and James

Reply:
I covered this in #2310

Reply:
And yes, to your question.